core/indigo-core/layout/pathway_layout.h

```diff
@@ -73,8 +75,18 @@ namespace indigo
                 if (reactantIdx != -1)
                 {
                     auto& mol = reaction.getMolecule(reactantIdx);
-                    // MoleculeLayout molLayout(mol, true);
-                    // molLayout.make();
+                    auto mean = mol.getBondsMeanLength();
+                    if (mean < MIN_BOND_MEAN)
```

Comment:
Maybe add check for bond count > 0?

Reply:
There is no reason for this check. Let's layout will decide what to do if bond count > 0 or not.